src/session/middleware/index.ts

```diff
@@ -15,18 +14,13 @@ declare global {
 
 export default async (req: Request, res: Response, next: NextFunction) => {
   const cookieId = req.cookies[COOKIE_NAME];
-  // if there is no cookie, we need to create a new session
-  if (!cookieId && activeFeature(SESSION_CREATE)) {
-    logger.info("No cookie found, creating new session");
-    const session = Session.newInstance();
-    session.setClientSignature(req.get("user-agent") || "", req.ip);
-    await redisService.saveSession(session);
 
-    req.chSession = session;
-    res.cookie(COOKIE_NAME, session.cookieId, {path: "/"});
-  } else {
+  if (cookieId) {
     logger.info("cookie found, loading session from redis: " + cookieId);
     req.chSession = await redisService.loadSession(cookieId);
+  }else {
```

Comment:
A space required before the 'else' I think. Did you run ts-lint? Thought this would pick it up...